src/ResourceManager/Resources/Commands.Resources/Properties/Resources.resx

```diff
@@ -420,4 +420,28 @@
   <data name="DefaultEndDateUsed" xml:space="preserve">
     <value>No value specified for -EndDate parameter; setting the value to one year after start date.</value>
   </data>
+  <data name="GroupWithDisplayNameDoesntExist" xml:space="preserve">
+    <value>Group with display name '{0}' does not exist.</value>
+  </data>
+  <data name="MultipleApplicationsWithDisplayNameFound" xml:space="preserve">
+    <value>More than one application found with display name '{0}'. Please use the Get-AzureRmADApplication cmdlet to get the object id of the desired application.</value>
```

Comment:
Get-AzAdApplication: Please go through these to remove references to 'AzureRm' in cmdlets